web/client/plugins/GeoStories.jsx

```diff
@@ -0,0 +1,132 @@
+/*
+* Copyright 2019, GeoSolutions Sas.
+* All rights reserved.
+*
+* This source code is licensed under the BSD-style license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+const React = require('react');
+const PropTypes = require('prop-types');
+const assign = require('object-assign');
+const {connect} = require('react-redux');
+const Message = require("../components/I18N/Message");
+const emptyState = require('../components/misc/enhancers/emptyState');
+
+const { setGeostoriesAvailable } = require('../actions/geostories');
+const {mapTypeSelector} = require('../selectors/maptype');
+const { userRoleSelector } = require('../selectors/security');
+const { isFeaturedMapsEnabled } = require('../selectors/featuredmaps');
+const { totalCountSelector } = require('../selectors/geostories');
+const {createSelector} = require('reselect');
+const { compose } = require('recompose');
+
+const GeostoryGrid = require('./geostories/GeostoriesGrid');
+const PaginationToolbar = require('./geostories/PaginationToolbar');
+const EmptyDashboardsView = require('./geostories/EmptyGeostoriesView');
+
+const geostoriesCountSelector = createSelector(
+    totalCountSelector,
+    count => ({ count })
+);
+
+
+class Dashboards extends React.Component {
+    static propTypes = {
+        mapType: PropTypes.string,
+        title: PropTypes.any,
+        onMount: PropTypes.func,
+        loadDashboards: PropTypes.func,
+        resources: PropTypes.array,
+        searchText: PropTypes.string,
+        mapsOptions: PropTypes.object,
+        colProps: PropTypes.object,
+        fluid: PropTypes.bool
+    };
+
+    static contextTypes = {
+        router: PropTypes.object
+    };
+
+    static defaultProps = {
+        mapType: "leaflet",
+        onMount: () => {},
+        loadDashboards: () => {},
+        fluid: false,
+        title: <h3><Message msgId="resources.geostories.titleNoCount" /></h3>,
+        mapsOptions: {start: 0, limit: 12},
```

Comment:
I would call it differently, like storyOptions, what about it @offtherailz ?

Reply:
Considering a future abstraction could be better resourcesOptions or shorted form resOption,
but has to be dane in dashboards too